libkbfs/folder_branch_ops.go

```diff
@@ -2204,6 +2218,20 @@ func (fbo *folderBranchOps) createEntryLocked(
 		newBlock = &FileBlock{}
 	}
 
+	// Passthrough journal writes temporarily.
+	if excl == WithExcl {
+		if jServer, err := GetJournalServer(fbo.config); err == nil {
+			if err := jServer.DisableWrites(ctx, fbo.id()); err != nil {
```

Comment:
I must be missing something, but why can't just `Disable` and `Enable` work? If the journal is completely empty, then it doesn't matter if we let reads pass through also, right?